script/openqa-bootstrap

```diff
@@ -76,6 +76,9 @@ if [[ $NAME = "openSUSE Leap" ]]; then
     zypper -n --gpg-auto-import-keys refresh
 fi
 
+# Ensure we have retry available to handle temporary package issues later
+command -v retry >/dev/null || zypper -n --gpg-auto-import-keys in retry
```

Comment:
```suggestion
command -v retry > /dev/null || zypper -n --gpg-auto-import-keys in retry
```

Reply:
Interesting. Didn't come up here when I ran the style check locally.

Reply:
Hum. Maybe it just got drowned in verbose output.